lib/modules/manager/swift/artifacts.ts

```diff
@@ -0,0 +1,107 @@
+import { TEMPORARY_ERROR } from '../../../constants/error-messages';
+import { logger } from '../../../logger';
+import { exec } from '../../../util/exec';
+import type { ExecOptions } from '../../../util/exec/types';
+import {
+  getSiblingFileName,
+  localPathExists,
+  readLocalFile,
+  writeLocalFile,
+} from '../../../util/fs';
+import type {
+  UpdateArtifact,
+  UpdateArtifactsConfig,
+  UpdateArtifactsResult,
+} from '../types';
+import { extractSwiftToolsVersion } from './util';
+
+async function swiftPackageResolve(
+  packageFile: string,
+  toolsVersion: string | null,
+  config: UpdateArtifactsConfig
+): Promise<void> {
+  const cmd = `swift package resolve`;
+  const versionRange =
+    toolsVersion === null ? toolsVersion : `>=${toolsVersion}`;
```

Comment:
that range makes no sense, as it's same as no range. renovate will use latest version then.

Reply:
let's not use a range for now. see other comments